mgr/src/cmds/helpers.cr

```diff
@@ -105,3 +107,23 @@ def yes(label)
   value = STDIN.gets(chomp: true).not_nil!
   ["yes", "y", "yy", "ok", "sure", "on"].includes?(value.strip.downcase)
 end
+
+def handle_json_output(data, args)
+  if args.json
+    if data && data != ""
+      puts data.to_pretty_json
+      exit 0
+    else
+      puts "{}".to_json
```

Comment:
This will print `"{}"` as string. Use `puts "{}"`

Reply:
When used `puts typeof("{}")` also prints as string, is it expected?